library/std/src/sys/unix/fd.rs

```diff
@@ -26,6 +27,35 @@ const READ_LIMIT: usize = c_int::MAX as usize - 1;
 #[cfg(not(target_os = "macos"))]
 const READ_LIMIT: usize = libc::ssize_t::MAX as usize;
 
+#[cfg(any(target_os = "linux", target_os = "macos"))]
+fn max_iov() -> usize {
```

Comment:
Does it include unix as _SC_IOV_MAX declared in POSIX 1: https://pubs.opengroup.org/onlinepubs/009695399/functions/sysconf.html ?

Reply:
I was not sure whether `#[cfg(unix)]` actually implies POSIX but as per @Amanieu's comment below, I will change this to purely rely on POSIX semantics.